package.json

```diff
@@ -1,4 +1,25 @@
 {
+  "name": "pwr2",
+  "description": "Docker image based on Ubuntu providing a base setup for a Phoenix+Webpack+Relay+React project, with some sugar and conventions to develop and build your own web applications.",
+  "main": "webpack.config.js",  
+  "version": "",
+  "directories": {
+    "test": "test"
+  },
+  "keywords": [],
+  "preferGlobal": true,
```

Comment:
sorry , I have little experience with these settings. Looking at the documentation thought I could use. I fix it?